pkg/queue/manager_test.go

```diff
@@ -160,6 +160,63 @@ func TestUpdateQueue(t *testing.T) {
 	}
 }
 
+func TestDeleteFromQueue(t *testing.T) {
```

Comment:
this test should be in `cluster_queue_best_effort_fifo.go`; the needs to explicitly create a fifo cq using newClusterQueueBestEffortFIFO, and you can directly check the inadmissibleWorkloads map

Reply:
update as advised.